career/linked-in.md

```diff
@@ -12,7 +12,7 @@ Example: 60 minutes
 
 Here are links to lessons that should be completed before this lesson:
 
-- [Resume writing](https://github.com/Techtonica/curriculum/blob/master/resume-writing/resume-writing.md)
+- [Resume writing](https://github.com/Techtonica/curriculum/blob/master/career/resume-writing.md)
```

Comment:
please turn this into a relative link

Reply:
I think that could be mentioned in the issue from next time.

Reply:
@Kundan28 that would not be realistic for me to know the link format of every document. I have linked in our [style guide](https://github.com/Techtonica/curriculum/blob/master/_templates/instructions.md#links) in the instructions so it is more clear.

Reply:
Oh, okay. @vegetabill